src/openrct2/world/TileElement.h

```diff
@@ -384,9 +384,9 @@ struct SmallSceneryElement : TileElementBase
 {
 private:
     ObjectEntryIndex entryIndex; // 4
-    uint8_t age;                 // 5
-    uint8_t colour_1;            // 6
-    uint8_t colour_2;            // 7
+    uint8_t age;                 // 6
+    uint8_t colour_1;            // 7
+    uint8_t colour_2;            // 8
```

Comment:
Why did you increment the address comments here?

Reply:
ObjectEntryIndex is 2 bytes, so I guess just correcting the comments.

Reply:
I think so too, they have no real meaning anymore.

Reply:
Yes, it was just a small correction of the comments.
I wont recommend to remove them, they are still relevant for the data padding and if you want to access the raw data from within a plugin.

Reply:
That sounds evil and shouldn't be done, all fields are explicitly exported, please remove the comments.

Reply:
No, currently there are a lot of fields that are not exported.
I personally have to rely on raw data access at several points in my plugins.

Anyhow, there is still the argument that it makes the padding more transparent.

Reply:
But if this is such a big issue maybe I should move it to a different PR.

Reply:
@Sadret just tell us whats missing and we can expose it properly.

Reply:
I planned to implement it myself, once I finished the scenery plugin.
At least for the scenery place and remove actions.